README.md

```diff
@@ -67,9 +67,16 @@ At this point you can run the Gibbon executable:
 
     $ cabal v2-exec -w ghc-9.0.1 gibbon -- -h
 
-If you'd like to run the testsuite, you can do so with:
+And add gibbon to PATH
+```
+$ export PATH = "{ ... }/gibbon/dist-newstyle/build/{architecture}/ghc-{version}/gibbon-0.2/x/gibbon/build/gibbon/gibbon:$PATH"
```

Comment:
It's possible that the dist-newstyle directory structure could change and we wouldn't notice. Did `v2-exec` not work for you?

Reply:
I think that path is just useful if you want to get the `gibbon` binary added to your PATH variable so its available on the terminal.

Reply:
Hey there! Just FYI: you don’t need to spell the path to the binary explicitly if you leverage `cabal`’s `list-bin` subcommand, e.g.

```
$ export PATH = "$(cabal list-bin gibbon):$PATH"
```